upgrade_test.py

```diff
@@ -162,89 +163,93 @@ def upgrade_node(self, node, upgrade_sstables=True):
-            backup_conf(node)
-            assert new_scylla_repo.startswith('http')
-            node.download_scylla_repo(new_scylla_repo)
-            # flush all memtables to SSTables
-            node.run_nodetool("drain", timeout=3600, coredump_on_timeout=True)
-            node.run_nodetool("snapshot")
-            node.stop_scylla_server(verify_down=False)
-
-            orig_is_enterprise = node.is_enterprise
-            if node.is_rhel_like():
-                result = node.remoter.run("sudo yum search scylla-enterprise 2>&1", ignore_status=True)
-                new_is_enterprise = bool('scylla-enterprise.x86_64' in result.stdout or
-                                         'No matches found' not in result.stdout)
-            else:
-                result = node.remoter.run("sudo apt-cache search scylla-enterprise", ignore_status=True)
-                new_is_enterprise = 'scylla-enterprise' in result.stdout
-
-            scylla_pkg = 'scylla-enterprise' if new_is_enterprise else 'scylla'
-            if orig_is_enterprise != new_is_enterprise:
-                self.upgrade_rollback_mode = 'reinstall'
-            ver_suffix = r'\*{}'.format(new_version) if new_version else ''
-            if self.upgrade_rollback_mode == 'reinstall':
+        with DbEventsFilter(db_event=DatabaseLogEvent.RUNTIME_ERROR,
+                            line="abort requested"):
+            # https://github.com/scylladb/scylla/issues/10447#issuecomment-1194155163
+            if upgrade_node_packages:
+                # update_scylla_packages
+                node.remoter.send_files(upgrade_node_packages, '/tmp/scylla', verbose=True)
+                # node.remoter.run('sudo yum update -y --skip-broken', connect_timeout=900)
+                node.remoter.run('sudo yum install python34-PyYAML -y')
+                # replace the packages
+                node.remoter.run(r'rpm -qa scylla\*')
+                # flush all memtables to SSTables
+                node.run_nodetool("drain", timeout=3600, coredump_on_timeout=True)
+                node.run_nodetool("snapshot")
+                node.stop_scylla_server()
+                # update *development* packages
+                node.remoter.run('sudo rpm -UvhR --oldpackage /tmp/scylla/*development*', ignore_status=True)
+                # and all the rest
+                node.remoter.run('sudo rpm -URvh --replacefiles /tmp/scylla/*.rpm | true')
+                node.remoter.run(r'rpm -qa scylla\*')
+            elif new_scylla_repo:
+                # backup the data
+                node.remoter.run('sudo cp /etc/scylla/scylla.yaml /etc/scylla/scylla.yaml-backup')
                 if node.is_rhel_like():
-                    node.remoter.run(r'sudo yum remove scylla\* -y')
-                    node.remoter.run('sudo yum install {}{} -y'.format(scylla_pkg, ver_suffix))
+                    node.remoter.run('sudo cp /etc/yum.repos.d/scylla.repo ~/scylla.repo-backup')
```

Comment:
What's this change?

Reply:
There was no change. Github just misunderstanding this whole thing.
All I did was inserting a tab in front of all of those rows